### PR TITLE
fix: horizontal text alignment for bound text when resizing

### LINKED ISSUE #5720: Text positioning in sticky notes does not honor Text Alignment when resizing the container
@ad1992 checking previous preview builds I can see this issue was introduced in #5710

https://user-images.githubusercontent.com/14358394/192335957-d30953bf-f17d-4a3e-9820-99203071b51b.mp4

### PR DESCRIPTION
**Pull request by zsviczian**
fixes #5720

### CODE REVIEW

**Review by ad1992 — COMMENTED** [review]
@zsviczian can you also add a test for this fix?

**Comment by zsviczian** [comment]
I am not sure how to add a test for this. I'd rather leave adding the test to you.

**Comment by ad1992** [comment]
I have added a test. @dwelle can you review the last [commit](https://github.com/excalidraw/excalidraw/pull/5721/commits/1adf7bd98ed73a280cf79e945980c2f496012671) ?

File: src/element/textWysiwyg.test.tsx [hunk 1]
```diff
@@ -813,5 +815,78 @@ describe("textWysiwyg", () => {
       ]);
       expect(text.containerId).toBe(null);
     });
+
+    it("should respect text alignment when resizing", async () => {
+      Keyboard.withModifierKeys({}, () => {
+        Keyboard.keyPress(KEYS.ENTER);
+      });
```
Comment (dwelle) [inline src/element/textWysiwyg.test.tsx:822]: no need for a `withModifierKeys` when no modifier is passed?

**Review by dwelle — APPROVED** [review]

File: src/element/textWysiwyg.test.tsx (hunk 1 above)
Comment (ad1992) [reply src/element/textWysiwyg.test.tsx]: right

**Comment by ad1992** [comment]
@zsviczian thanks for the fix!

**Pull request merged**